packages/@react-aria/utils/src/isFocusable.ts

```diff
@@ -1,3 +1,17 @@
+/*
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {isElementVisible} from './isElementVisible';
+
```

Comment:
while we're in this file, want to add https://github.com/adobe/react-spectrum/issues/8226

Reply:
I was confused by the comment - which makes it sound like `.focus` is not yet supported? https://github.com/WICG/PEPC/issues/50

Reply:
o, interesting, and seems maybe like it won't ever be. don't worry about it in this PR then. we'll have to figure out what to do with it.